### ISSUE TITLE
__TS__CloneDescriptor not included in source file

### ISSUE THREAD

**Issue by Zytharian** [TypeScriptToLua/TypeScriptToLua#1073]
On playground (TSTL v0.41.0) with this code:
```
class Taco {
    public get isApple(): boolean {
        return false;
    }
}
```

I get the error `attempt to call a nil value (global '__TS__CloneDescriptor')` because the source of that function doesn't get included in the compiled Lua file.

**Comment by Perryvw** [comment]
Seems like a missing lualib dependency. Should not be a problem if you don't use inlined lualib (default settings just require the bundle), but this should be fixed anyway.

**Comment by Perryvw** [comment]
Fixed with #1075
